tests/resources/test_resources.py

```diff
@@ -3,6 +3,7 @@
 import pytest
 from taurus.entity.bounds.real_bounds import RealBounds
 
+from citrine.informatics.scorers import MLIScorer, MEIScorer
```

Comment:
this is already taken care of in tests/informatics/test_informatics.py

Reply:
@kyle-laplante nice - i'll bump it out

Reply:
Removed in 10de9e7b57fa266b9b96a0ed0df9e3a41959079a